packages/ods/src/style/_overlay.scss

```diff
@@ -0,0 +1,19 @@
+$ods-overlay-z-index: 99;
+
+@mixin ods-tooltip() {
```

Comment:
ods-overlay

Reply:
not really, this style is the one for a tooltip (and a popover), but there will be other overlay who may look different, for example we may use the overlay code for the select.